design/decisions/why-license/index.qmd

```diff
@@ -0,0 +1,67 @@
+---
+title: "Why MIT License"
+description: |
+  Open and permissive licensing of copyrighted content is a way to increase use of a software product, ease collaboration and re-use, and share intellectual ideas with others. The MIT License is a very permissive copyright license.
+author: "Kristiane Beicher"
+date: "2023-04-01"
+date-modified: last-modified
+categories:
+  - licensing
+  - copyright
+  - software-architecture
+---
+
+## Context and Problem Statement
+
+When developing a new piece of software it is important to consider the following question as early as possible.
+
+> What kind of re-use will we ultimately allow other developers to make of our code/product?
```

Comment:
Maybe just "product" or "software", if we by the "Seedcase Product" or "Seedcase Software" mean the software we will develop? Refers to #181 about what we mean when we mention "Product", "Seedcase", etc.